lib/salesforce_ar_sync/jobs/delete_object_job.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module SalesforceArSync
+  class DeleteObjectJob < BaseJob
+    def perform(klass, sobject)
+      klass = prepare_class(klass)
+      klass.delete_object(sobject)
```

Comment:
You probably don't need the variable here
```suggestion
      prepare_class(klass).delete_object(sobject)
```